de.metas.adempiere.adempiere/base/src/main/java/de/metas/event/impl/EventBus.java

```diff
@@ -190,20 +201,16 @@ public void postEvent(@NonNull final Event event)
 			return;
 		}
 
+		if (event.isStoreEvent() && event.isLocalEvent())
```

Comment:
```suggestion
		// as long as we have just one common event-log-DB, we store events only on the machine they were created on, in order to avoid duplicates.
		if (event.isStoreEvent() && event.isLocalEvent())
```